forward_engineering/helpers/generalHelper.js

```diff
@@ -83,11 +83,35 @@ const getDefaultName = () => {
 	return defaultName;
 };
 
+const convertName = schema => {
+	_ = dependencies.lodash;
+
+	const nameProperties = ['typeName', 'code', 'name', 'displayName'];
+	const nameKey = nameProperties.find(key => schema[key]);
+	if (!nameKey) {
+		return schema;
+	}
+
+	return { ..._.omit(schema, nameProperties), name: prepareName(schema[nameKey])};
+};
+
+const compareSchemasByStructure = (schema1, schema2) => {
+	const propertiesToCompare = ['type', 'name', 'fields', 'items', 'values', 'logicalType', 'precision', 'scale', 'size'];
+
+	return _.isEqualWith(schema1, schema2, (schema1, schema2, key) => {
+		if (!propertiesToCompare.includes(key)) {
```

Comment:
if this handler returns "undefined" - comparison will be handled by default lodash handler